detailed_search_autonomy.py

```diff
@@ -59,11 +59,13 @@ def xbee_callback(message, autonomyToCV):
 
 def orbit_poi(vehicle, poi, configs):
     waypoints = []  # waypoints in LLA
+    altitude = configs["altitude"]
     poi_scan_altitude = configs["detailed_search_specific"]["poi_scan_altitude"]
     waypoint_tolerance = configs["waypoint_tolerance"]
     radius = configs["radius"]  # radius of circle travelled
     orbit_number = configs["orbit_number"]  # how many times we repeat orbit
-    x, y, z = conversions.geodetic2ecef(poi.lat, poi.lon, configs["altitude"])  # LLA -> ECEF
+    x, y, z = geodetic2ecef(poi.lat, poi.lon, poi_scan_altitude)  # LLA -> ECEF
+    n, e, d = ecef2ned(x, y, z, poi.lat, poi.lon, poi_scan_altitude)  # ECEF -> NED
```

Comment:
Should these changes be on this PR? or is this different...?

Reply:
these were from a pull